frontend/src/views/web/challenge/leaderboard.html

```diff
@@ -28,28 +28,54 @@
+                                            {{key}}
+                                            <span ng-show="$parent.challenge.sortColumn == 'number' && $parent.challenge.columnIndexSort == '{{$index}}'">
+                                                <span ng-show="!$parent.challenge.reverseSort">&uarr;</span>
+                                                <span ng-show="$parent.challenge.reverseSort">&darr;</span>
+                                            </span>
+                                        </a>
+                                    </td>
+                                    <td class="align-right" data-field="submission_time">
+                                        <a href="#" ng-click="challenge.sortColumn = 'date'; challenge.reverseSort = !challenge.reverseSort">
+                                            Last Submission at
+                                            <span ng-show="challenge.sortColumn == 'date'">
+                                                <span ng-show="!challenge.reverseSort" class="glyphicon glyphicon-chevron-up">&uarr;</span>
+                                                <span ng-show="challenge.reverseSort" class="glyphicon glyphicon-chevron-down">&darr;</span>
+                                            </span>
+                                        </a>
+                                    </td>
+                                </tr>
                                 </thead>
                                 <tbody>
-                                    <tr ng-repeat="key in challenge.leaderboard">
-                                        <td class="align-left">{{$index+1}}</td>
-                                        <td class="font-weight-b align-left">{{key.submission__participant_team__team_name}}</td>
-                                        <td class="align-right" ng-repeat="score in key.result track by $index" >{{score | number : 2}}</td>
-                                        <td class="align-right">
-                                        {{key.submission__submitted_at | number: 0}}&nbsp;{{key.timeSpan}} ago
-                                        </td>
-                                    </tr>
+                                <tr ng-repeat="key in challenge.leaderboard|orderBy:challenge.sortFunction:challenge.reverseSort">
+                                    <td class="align-left">{{$index+1}}</td>
+                                    <td class="font-weight-b align-left">{{key.submission__participant_team__team_name}}</td>
+                                    <td class="align-right" ng-repeat="score in key.result track by $index" >{{score | number : 2}}</td>
+                                    <td class="align-right">{{ key.submission__submitted_at | date:'medium'}}&nbsp;{{key.timeSpan}} ago</td>
```

Comment:
there seems to be a small error here. Can you please replace the line 77 and 78 with this code:

```
<td class="align-right">{{key.submission__submitted_at | number: 0}}&nbsp;{{key.timeSpan}} ago</td>
```